esbuild.mjs

```diff
@@ -1,6 +1,8 @@
 import * as path from "path";
+import { fileURLToPath } from "url";
 import * as esbuild from "esbuild";
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
This is clever!

General question, would you consider using `import.meta.url` a better way to get which module was loaded compared to throwing an error end search the stacktrace?

Reply:
It's an alternative to using `__dirname` which doesn't work in .mjs files. I don't know if it's much better though 😋